frontend/packages/ceph-storage-plugin/src/plugin.ts

```diff
@@ -72,8 +72,8 @@ const plugin: Plugin<ConsumedExtensions> = [
   {
     type: 'FeatureFlag/Action',
     properties: {
-      flag: OCS_VERSION_4_5_FLAG,
-      detect: detectOCSVersion45AndAbove,
+      flag: OCS_SNAPSHOT,
```

Comment:
@cloudbehl This value defines a flag. We cannot set up a new flag inside the detection method. Right now we cannot pass multiple flags to this particular property. We need to change the type signature as well as add logic to parse array and set flags.

Reply:
I understood the problem. Do we want to go in the same way that @a2batic  has done? 

To have one function matching to one feature flag action?

Reply:
I have raised a PR for that, if it does not go in then we can go the way @a2batic has done for now and refactor it in 4.6 for sure.